From 6b8054793c914f41c88af9d1c26a113bf78ff8f4 Mon Sep 17 00:00:00 2001
From: Ranjan Dutta <ranjan.dutta@intel.com>
Date: Sun, 20 Nov 2022 20:20:40 +0800
Subject: [PATCH 17/56] Revert "gpu: lontium-lt9611: Fix NULL pointer
 dereference in lt9611_connector_init()"

This reverts commit 3959e8faf8bf6bea619e8856c736db64e6eced37.
---
 drivers/gpu/drm/bridge/lontium-lt9611.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/bridge/lontium-lt9611.c b/drivers/gpu/drm/bridge/lontium-lt9611.c
index 1dcc28a4d853..29b1ce2140ab 100644
--- a/drivers/gpu/drm/bridge/lontium-lt9611.c
+++ b/drivers/gpu/drm/bridge/lontium-lt9611.c
@@ -816,14 +816,13 @@ static int lt9611_connector_init(struct drm_bridge *bridge, struct lt9611 *lt961
 
 	drm_connector_helper_add(&lt9611->connector,
 				 &lt9611_bridge_connector_helper_funcs);
+	drm_connector_attach_encoder(&lt9611->connector, bridge->encoder);
 
 	if (!bridge->encoder) {
 		DRM_ERROR("Parent encoder object not found");
 		return -ENODEV;
 	}
 
-	drm_connector_attach_encoder(&lt9611->connector, bridge->encoder);
-
 	return 0;
 }
 
-- 
2.25.1

